src/Microsoft.AspNetCore.Hosting/Internal/WebHostOptions.cs

```diff
@@ -30,6 +30,12 @@ public WebHostOptions(IConfiguration configuration, string applicationNameFallba
             WebRoot = configuration[WebHostDefaults.WebRootKey];
             ContentRootPath = configuration[WebHostDefaults.ContentRootKey];
             PreventHostingStartup = WebHostUtilities.ParseBool(configuration, WebHostDefaults.PreventHostingStartupKey);
+
+            if (!string.IsNullOrEmpty(configuration[WebHostDefaults.WriteStatusMessages]))
```

Comment:
Just to be sure, is PreventHostingStartup a required part of configuration? Just trying to understand why we don't do the same check for it.

Reply:
Is it because ParseBool returns false if the value is missing and would override the defaults? Some claim all bool defaults should be false anyways, and you name the property accordingly. SuppressConsoleMessages?

Reply:
Yea I thought about that but I supposed at some point, we might consolidate the web host and generic host. It would be rather odd that one has the option to suppress console messages while the other enables it. We could make both suppress? Or we can just live with the discrepancy.